Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityProjectConfigurator.cs

```diff
@@ -36,6 +39,15 @@ public enum Configurations
 #if UNITY_2019_3_OR_NEWER
             EyeTrackingCapability,
 #endif
+
+            // Android Settings
+            AndroidMultiThreadedRendering,
```

Comment:
We should add remark that this enumeration should not be saved to disk because exact values are subject to change?
Basically if we add more WSA capabilities it'll be weird to append at the end of the list. So either we accept that this is only a runtime interface or we should bucket the platforms like below

// wsa
SpatialPerceptionCapability = 1000,
//android
AndroidMultiThreadedRendering = 2000,
//ios
IOSMinOSVersion = 3000

Reply:
i can go for bucketing.